client/blocks/plugins-update-manager/index.tsx

```diff
@@ -39,14 +42,22 @@ export const PluginsUpdateManager = ( props: Props ) => {
 				) }
 			</NavigationHeader>
 
-			{ context === 'list' && (
-				<ScheduleList
-					siteSlug={ siteSlug }
-					onNavBack={ onNavBack }
-					onCreateNewSchedule={ onCreateNewSchedule }
-				/>
-			) }
-			{ context === 'create' && <ScheduleCreate siteSlug={ siteSlug } onNavBack={ onNavBack } /> }
+			{
+				{
+					list: (
+						<ScheduleList
+							siteSlug={ siteSlug }
+							onNavBack={ onNavBack }
+							onCreateNewSchedule={ onCreateNewSchedule }
+							onEditSchedule={ onEditSchedule }
+						/>
+					),
+					create: <ScheduleCreate siteSlug={ siteSlug } onNavBack={ onNavBack } />,
+					edit: (
+						<ScheduleEdit siteSlug={ siteSlug } scheduleId={ scheduleId } onNavBack={ onNavBack } />
+					),
+				}[ context ]
```

Comment:
Neat, I think this is the first time I'm seeing this kind of usage in JSX. Cleaner than the conditionals!

Reply:
Yeah, I like it as well. Much better than the conditions.